cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/util/Env.java

```diff
@@ -36,6 +40,17 @@ public static String get(@NotNull final String key) {
     }
 
     @NotNull
+    public static Map<String, String> optionsStart(@NotNull final String prefix, final boolean trimPrefix) {
+        final Map<String, String> options = newHashMap();
+
+        for (final String key : System.getenv().keySet())
+            if (key.startsWith(prefix))
+                options.put(trimPrefix ? key.substring(prefix.length()) : key, System.getenv(key));
```

Comment:
Rather than looking up the value from the system again, change your iteration to be `entrySet` then just use `entry.getValue`